### ISSUE TITLE
UpgradePending during operator installation on OCP 4.3

### ISSUE THREAD

**Issue by alvarolop** [kiegroup/kogito-cloud-operator#535]
Hello,

I am trying to install the latest version of Kogito Operator on OCP 4.3.3. I am selecting the following subscription:
```
spec:
  channel: alpha
  installPlanApproval: Automatic
  name: kogito-operator
  source: community-operators
  sourceNamespace: openshift-marketplace
  startingCSV: kogito-operator.v0.14.0
```

The result is that the operator is stuck without installing. This is the output of the status section:
```
status:
  catalogHealth:
    - catalogSourceRef:
        apiVersion: operators.coreos.com/v1alpha1
        kind: CatalogSource
        name: certified-operators
        namespace: openshift-marketplace
        resourceVersion: '123542'
        uid: 611222b3-d44e-4e39-966f-6c09bff3618a
      healthy: true
      lastUpdated: '2020-08-24T22:12:14Z'
    - catalogSourceRef:
        apiVersion: operators.coreos.com/v1alpha1
        kind: CatalogSource
        name: community-operators
        namespace: openshift-marketplace
        resourceVersion: '123540'
        uid: 7e97034e-98f5-4509-b648-96ff9ae770be
      healthy: true
      lastUpdated: '2020-08-24T22:12:14Z'
    - catalogSourceRef:
        apiVersion: operators.coreos.com/v1alpha1
        kind: CatalogSource
        name: redhat-operators
        namespace: openshift-marketplace
        resourceVersion: '123541'
        uid: 286c8708-3003-4934-a572-fb13701b63cb
      healthy: true
      lastUpdated: '2020-08-24T22:12:14Z'
  conditions:
    - lastTransitionTime: '2020-08-24T22:12:14Z'
      message: all available catalogsources are healthy
      reason: AllCatalogSourcesHealthy
      status: 'False'
      type: CatalogSourcesUnhealthy
    - lastTransitionTime: '2020-08-24T22:12:17Z'
      reason: InstallComponentFailed
      status: 'True'
      type: InstallPlanFailed
  currentCSV: kogito-operator.v0.14.0
  installPlanRef:
    apiVersion: operators.coreos.com/v1alpha1
    kind: InstallPlan
    name: install-hwwtm
    namespace: alvaro-kogito
    resourceVersion: '161220'
    uid: 0e784f86-9ed2-427f-97ac-c31c084b7864
  installplan:
    apiVersion: operators.coreos.com/v1alpha1
    kind: InstallPlan
    name: install-hwwtm
    uuid: 0e784f86-9ed2-427f-97ac-c31c084b7864
  lastUpdated: '2020-08-24T22:12:17Z'
  state: UpgradePending
```

Installing version 0.9 works, but this version is too old for me. Isn't there an intermediate stable channel between 0.9 and 0.14?

Thank you for your work.

Best regards,

**Comment by ricardozanini** [comment]
Hi thanks for reaching out!

We've tested with OCP 4.5 and we haven't seen this error in this version. We can take a look on OCP 4.3 later, but in the meantime could you please try to change the Infinispan channel to **1.1.x**? There are some known incompatibility issues that should be solved on 0.15 when we remove the dependencies from third party operators.

Also, it's worth mentioning that there are some changes in the OLM and how to install the operators in the cluster in this new version that we are targeting. 

If changing Infinispan channel fails, could you please try installing it manually via our [installation script](https://github.com/kiegroup/kogito-cloud-operator/blob/v0.14.0/hack/install.sh)? You would need to download the latest source from the [release page](https://github.com/kiegroup/kogito-cloud-operator/releases/tag/v0.14.0). Thanks!

**Comment by alvarolop** [comment]
Hello Ricardo, 

Sorry for not contacting you sooner. I have tried several approaches, but none of them worked. How did you know that I was installing Infinispan 2.X? :) I tried with Infinispan 1.1.x.

This week I tried again in the same OCP 4.3.3, but the result was different. I decided not to install Infinispan and Strimzi. Kogito operator managed to install Infinispan and Strimzi correctly but stuck installing itself (Similar as what happened in my previous message).

I have managed to get limited access to an OCP 4.5.6 cluster and everything is working like a charm. Thank you for your help.
